common/zext_wrapper.py

```diff
@@ -6,6 +6,11 @@ class ZextWrapper(generator.Generator):
     def __init__(self, in_width, out_width):
         super().__init__()
 
+        if out_width <= in_width:
+            raise ValueError(f"input width must be greater than output width "
```

Comment:
This error message should be "output width must be greater than input width", right?

Reply:
Done